WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/TopTabView.swift

```diff
@@ -148,72 +154,72 @@ struct TopTabView<Content: View>: View {
                 Divider()
             }
 
-            // Display all the tabs in an HStack, each tab the same width as the TopTabView
```

Comment:
Looks like a big code change but I just put the content part of the code in one `if showContent` statement